### ISSUE TITLE
模拟器选择视频崩溃

### ISSUE THREAD

**Issue by Homin818** [banchichen/TZImagePickerController#536]
*** Assertion failure in NSArray *_UIAccessibilityFocusedElements(), @:105553133519680.
|NSObjectAccessibility.m:7526 NSArray *_UIAccessibilityFocusedElements()|Could not execute on the main thread

TZVideoPlayerController.m:63

    [[TZImageManager manager] getVideoWithAsset:_model.asset completion:^(AVPlayerItem *playerItem, NSDictionary *info) {
        dispatch_async(dispatch_get_main_queue(), ^{
            _player = [AVPlayer playerWithPlayerItem:playerItem];
            _playerLayer = [AVPlayerLayer playerLayerWithPlayer:_player];
            _playerLayer.frame = self.view.bounds;
            [self.view.layer addSublayer:_playerLayer];
            [self addProgressObserver];
            [self configPlayButton];
            [self configBottomToolBar];
            [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(pausePlayerAndShowNaviBar) name:AVPlayerItemDidPlayToEndTimeNotification object:_player.currentItem];
        });
    }];

**Comment by banchichen** [comment]
你好，我这没复现出来...你是在我的Demo里复现的吗？是每个视频都崩溃，还是某些会崩溃呢？
看日志是被断言搞崩了，在主线程执行了不应该执行的操作...
